### DISCUSSION TITLE
Error CS0234 in Derived types mappers: The type or namespace name 'ArgumentException' does not exist

### DISCUSSION THREAD

**Discussion by zhant** [riok/mapperly#1021] [Q&A]
Hey! 

I'm not sure this is a bug but probably a misconfiguration as I am a newbie in Mapperly.

In _some_ project configurations I have an issue with Derived types mappers. Trying to figure out the reason I tried to copy a sample from https://mapperly.riok.app/docs/configuration/derived-type-mapping/ 
1) a new solution/project .net 6 - everything is fine.
2) existing project with .net 6. 
the latter case I receive 
`ModelMapper.g.cs(13,39): Error CS0234 : The type or namespace name 'ArgumentException' does not exist in the namespace '[MyProject.RootNamespace].System' (are you missing an assembly reference?)`

flat mapping is fine in both cases.

What can cause this? 

Thank you so much!

Ante

**Comment by latonz** [comment]
I can't reproduce, can you create a minimal reproducible git repository?
